openlibrary/templates/books/add.html

```diff
@@ -68,6 +68,8 @@ <h1>$_("Add a book to Open Library")</h1>
                 <div class="label">$_("Since you are not logged in, please satisfy the reCAPTCHA below.")</div>
                 <div class="g-recaptcha" data-sitekey="$:recaptcha.public_key"></div>
             </div>
+        
+        $:macros.EditButtons(comment=work.comment_)
```

Comment:
This should be removed, and the disclaimer should be inside of the next div.

Reply:
Thanks for the screenshot showing the mistake. I have made a _new_ pull request #5604 with the mistakes corrected and taking your suggestions into consideration.